servers/src/main/java/tachyon/worker/netty/protocol/RPCMessage.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker.netty.protocol;
+
+import com.google.common.primitives.Ints;
+import com.google.common.primitives.Longs;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.handler.codec.ByteToMessageDecoder;
+import io.netty.handler.codec.LengthFieldBasedFrameDecoder;
+
+import tachyon.worker.netty.protocol.buffer.DataBuffer;
+
+public abstract class RPCMessage implements EncodedMessage {
```

Comment:
NIT: it seems this class is really specific to Block Request/response messages. Is it better to name it after RPCBlockMessage ?